tests/inputs/gen_rar.sh

```
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -euo pipefail
+cd "$(dirname "$0")"
+d=$(mktemp -d)
+trap 'rm -rf "$d"' EXIT
+git clone --depth 1 --single-branch https://github.com/ssokolow/rar-test-files.git "$d"
+cp "$d/build/"* .
+echo "Copied $(ls -1 "$d/build/"* | wc -l) test files"
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)